src/Sia.Data.Incident/Migrations/MigrationConfigurationLoader.cs

```diff
@@ -15,12 +15,11 @@ public MigrationConfigurationLoader()
 
         public IncidentContext CreateDbContext(string[] args)
         {
-            IConfigurationRoot configuration = new ConfigurationBuilder().AddUserSecrets<MigrationConfigurationLoader>().Build();
-            var builder = new DbContextOptionsBuilder<IncidentContext>();
-
-            var connectionString = configuration.GetConnectionString("ppe");
-
-            builder.UseSqlServer(connectionString);
+            IConfigurationRoot configuration = new ConfigurationBuilder()
```

Comment:
Because this is only needed during initial database setup or apply a migration, I think this is pretty good! Running Gateway gets connection string from ServiceStartup. We will make changes there to get connection string from KeyVault for container deployment.